.github/workflows/build-beta.yml

```diff
@@ -0,0 +1,74 @@
+name: Build beta
+
+on:
+  workflow_call:
+
+jobs:
+  build-beta:
+    name: Build beta
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # This is needed to get the latest commit message instead of the merge commit message
```

Comment:
Include in the comment above that `github.event.pull_request.head.sha` is what changes when `fetch-depth: 0` is added